source/_components/utility_meter.markdown

```diff
@@ -128,3 +129,37 @@ automation:
     - service: utility_meter.next_tariff
       entity_id: utility_meter.monthly_energy
 ``` 
+# {% linkable_title Advanced Configuration for DSMR users %}
+
+When using the [DSMR component](https://www.home-assistant.io/components/sensor.dsmr/) to get data from the utility meter, each tariff (peak and off-peak) has a separate sensor. Additionally, there is a separate sensor for gas consumption. The meter switches automatically between tariffs, so an automation is not necessary in this case. But, you do have to setup a few more instances of the `utility_meter` component. 
+
+If you want to create a daily and monthly sensor for each tariff, you have to track separate sensors:
+- `sensor.power_consumption_low` for off-peak power 
+- `sensor.power_consumption_normal` for peak power
+- `sensor.gas_consumption` for gas consumption
+
+So, tracking daily and monthly consumption for each sensor, will require setting up 6 entries under the `utility_meter` component.
+
+```yaml
+utility_meter:
+  daily_power_offpeak:
+    source: sensor.power_consumption_low
+    cycle: daily 
+  daily_power_peak:
+    source: sensor.power_consumption_normal
+    cycle: daily
+  daily_gas:
+    source: sensor.gas_consumption
+    cycle: daily 
+  monthly_power_offpeak:
+    source: sensor.power_consumption_low
+    cycle: monthly
+  monthly_power_peak:
+    source: sensor.power_consumption_normal
+    cycle: monthly 
+  monthly_gas:
+    source: sensor.gas_consumption
+    cycle: monthly
+```
+
+6 different sensors will be created, 2 per sensor; `sensor.daily_energy_peak`, `sensor.daily_energy_offpeak`, `sensor.daily_gas` to track daily consumption and `sensor.monthly_energy_peak`, `sensor.monthly_energy_offpeak`, `sensor.monthly_gas` for monthly cycles.
```

Comment:
I think it is obvious that 6 sensores will be created since you defined 6 entities.

Reply:
makes sense, removed the last paragraph.